src/Okta.Idx.Sdk/AuthenticationStatus.cs

```diff
@@ -55,6 +55,11 @@ public enum AuthenticationStatus
         /// </summary>
         AwaitingChallengeAuthenticatorSelection,
 
+        /// <summary>
+        /// Waiting for push notification response.
+        /// </summary>
+        AwaitingChallengePollResponse,
```

Comment:
Can you please add this new status in the readme? there's a section for authentication statuses.